### DISCUSSION TITLE
Airbyte connector for supabase

### DISCUSSION THREAD

**Discussion by kish2011** [supabase/supabase#938] [Questions] [answered]
I am planning to Airbyte Postgres(https://airbyte.io/destination/database/postgres-2/) connector in my ETL. Can I use supabase as it also uses Postgres internally? Is there any restriction for it?

**Comment by steve-chavez** [comment, marked as answer]
Hey @kish2011,

There's no restriction since we're plain PostgreSQL under the hood. As long as the external service uses a pg connection string(which you'll find on `/settings/database`) you'll be able to use it.
